src/commands/setup.yml

```diff
@@ -8,9 +8,13 @@ parameters:
 
 steps:
   - run:
+      when: always
```

Comment:
What does the `always` do for us?

Reply:
Also, would be cool if we could chuck this thing into a workspace and have that load so as to reduce the fetching and unpacking we have to do

Reply:
> What does the always do for us?

We might be running this as part of a code path that had failed (if failed show this notification, else show that one). We always (he) want to run this step